src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -21,10 +21,15 @@ public sealed class CSharpParseOptions : ParseOptions, IEquatable<CSharpParseOpt
         private ImmutableDictionary<string, string> _features;
 
         /// <summary>
-        /// Gets the language version.
+        /// Gets the effective language version.
         /// </summary>
         public LanguageVersion LanguageVersion { get; private set; }
 
+        /// <summary>
+        /// Gets the specified language version.
+        /// </summary>
+        public LanguageVersion SpecifiedLanguageVersion { get; private set; }
```

Comment:
`SpecifiedLanguageVersion` is the value of the `LanguageVersion` enum that was specified in the call to the constructor, or modified using the `With` method, or provided on the command line.

`LanguageVersion` is the effective language version that the compiler uses to select which language rules to apply to the program.